source/core.py

```diff
@@ -69,6 +69,53 @@ def __bool__(self):
 _shuttingDownFlagLock = threading.Lock()
 
 
+def _showAddonsWarnings() -> None:

```

Comment:
Since we are displaying an error message:

```suggestion
def _showAddonsUpdateErrors() -> None:
```

or only:

```suggestion
def _showAddonsErrors() -> None:
```